compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -273,7 +274,9 @@ internal actual class ComposeWindow : UIViewController {
                 point = CGPointMake(0.0, 0.0),
                 toCoordinateSpace = UIScreen.mainScreen.coordinateSpace()
             )
-        return topLeftPoint.useContents { Offset(x.toFloat(), y.toFloat()) }
+        return with(density) {
+            topLeftPoint.useContents { DpOffset(x.dp, y.dp).toOffset() }
+        }
```

Comment:
Previously getTopLeftOffset was return wrong value (because it contains points in Float).
Now, getTopLeftOffset will return pixel offset.  Density helps to convert values.